website/docs: remove outdated info (#8552)
goauthentik/authentik: website/docs/outposts/embedded/embedded.mdx

You can access the embedded outpost on the same ports as authentik itself, 9000 and 9443.

> The embedded outpost is not required unless you are using the Proxy provider; if it doesn't make sense to use it in your deployment you can simply ignore it.

Reviewer: ```suggestion
If the embedded outpost doesn't make sense for your deployment, you can simply ignore it.
```
